pkg/charts/values.go

```diff
@@ -74,6 +75,12 @@ type (
 	// ConfigJSONs is the JSON encoding of the Linkerd configuration
 	ConfigJSONs struct{ Global, Proxy, Install string }
 
+	// ControlPlaneTrace contains fields to enable Control Plane Components Tracing
+	ControlPlaneTrace struct {
+		Enabled    bool
+		ProxyTrace *Trace
```

Comment:
why is this field called ProxyTrace?  This controls traces emitted directly from the controller, right?

Reply:
Yes, but this implies the svcAccount and svcAddress needed by the Control-plane proxies.